Resources/Prototypes/_Sunrise/Language/PredatorLanguage.yml

```
@@ -0,0 +1,52 @@
+    - qhe
+    - qhu
+    - qhei
+    - qhii
+    - qhra
+    - xa
+    - xe
+    - xi
+    - xei
+    - xae
+    - ju
+    - jae
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)